### PR TITLE
New package: python3-liblarch-3.0.1

### LINKED ISSUE #27373: [Package Request] python3-liblarch
Homepage: [https://github.com/getting-things-gnome/liblarch](https://github.com/getting-things-gnome/liblarch)
Description: Liblarch is a python library built to easily handle data structure such are lists, trees and acyclic graphs
Wiki: [https://wiki.gnome.org/Projects/liblarch](https://wiki.gnome.org/Projects/liblarch)

### PR DESCRIPTION
**Pull request by Bleznudd**
Should close #27373

### CODE REVIEW

File: srcpkgs/python3-liblarch/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+wrksrc=liblarch-${version}
+build_style=python3-module
+makedepends="python3-setuptools"
+depends="python3 python3-gobject gtk+3"
+short_desc="Library to handle data structures such as trees and acyclic graphs"
+maintainer="Bleznudd <bleznudd@pm.me>"
+license="LGPL-3.0-only"
+homepage="https://github.com/getting-things-gnome/liblarch"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=3a866cc43873454b88ae5526207555aca1f0e16e3120577d6135ea13db98ca68
```
Comment (kawaiiamber) [inline srcpkgs/python3-liblarch/template:14]: On line 16, add
```sh
if [ "${CROSS_BUILD}" ]; then
	hostmakedepends="python3"
fi
```
as `python3-setuptools` is not on crossbuild.

**Review by kawaiiamber — CHANGES_REQUESTED** [review]
Changes to pass cross build.

File: srcpkgs/python3-liblarch/template (hunk 1 above)
Comment (Bleznudd) [reply srcpkgs/python3-liblarch/template:14]: Many thanks, it seems to be working ok now

**Comment by Bleznudd** [comment]
Closing to move this commit in the [#27883](https://github.com/void-linux/void-packages/pull/27883) PR

**Pull request closed without merging**